reference/api/search.md

```diff
@@ -432,24 +441,24 @@ To get only the `overview` and `title` fields, set `attributesToRetrieve` to `["
 ### Attributes to crop
 
 **Parameter**: `attributesToCrop`
-**Expected value**: an array of attribute or `["*"]`
+**Expected value**: an array of attributes or `["*"]`
 **Default value**: `null`
 
-Crops the selected attributes' values in the returned results to the length indicated by the [`cropLength`](#crop-length) parameter.
+Crops the selected fields to the length indicated by the [`cropLength`](#crop-length) parameter.
 
-When this parameter is set, a field called `_formatted` will be added to `hits`. The cropped version of each document will be available there.
+By default, crop boundaries are marked by the ellipsis character (`…`). You can change this by using the [`cropMarker`](#crop-marker) search parameter.
 
-Optionally, you can indicate a custom crop length for any of the listed attributes: `attributesToCrop=["attributeNameA:25", "attributeNameB:150"]`. The custom crop length set in this way has priority over the `cropLength` parameter.
+When `attributesToCrop` is set, each returned document contains an extra field called `_formatted`. This object contains the cropped version of the selected attributes.
 
-Instead of supplying individual `attributes`, you can provide `["*"]` as a value: `attributesToCrop=["*"]`. This will crop the values of all attributes present in [`attributesToRetrieve`](#attributes-to-retrieve).
+Optionally, you can indicate a custom crop length for any attributes given to `attributesToCrop`: `attributesToCrop=["attributeNameA:5", "attributeNameB:9"]`. If configured, these values have priority over `cropLength`.
 
-**Cropping starts at the first occurrence of the search query**. It only keeps `cropLength` characters on each side of the first match, rounded to match word boundaries.
+Instead of supplying individual attributes, you can provide `["*"]` as a wildcard: `attributesToCrop=["*"]`. This causes `_formatted` to include the cropped values of all attributes present in [`displayedAttributes`](/learn/configuration/settings.md#displayed-attributes).
 
-If no query word is present in the cropped field, the crop will start from the first word.
+**Cropping starts at the first occurrence of any one of the terms present in the search query.** If Meilisearch does not find any query terms in a field, cropping begins at the first word in that field.
```

Comment:
> Cropping starts at the first occurrence of any one of the terms

The cropping does not start at the first occurrence. It starts around in a smartcroping way. 

Example in the code sample: 

query is `shifu` and output is `"…this year Shifu informs Po…"`

@dichotommy could enlighten on this behavior!

Reply:
Both what @guimachiavelli  wrote and what @bidoubiwa wrote are true, I think.

Saying that "cropping **starts** at the first occurrence of any query term" (my emphasis) invites a bit of confusion; might be more clear to say that Meilisearch crops **around** the first occurrence of any query term in a value.